### PR TITLE
adding profile to public/data

### PR DESCRIPTION
**Pull request by ahmedsilat44**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1803"></a>

### CODE REVIEW

File: public/data/ahmedsilat44.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "title": "",
+      "date": "",
+      "icon": "",
+      "color": "",
+      "description": "",
+      "url": ""
+    }
+  ]
+}
```
Comment (schmelto) [inline public/data/ahmedsilat44.json:28]:
```suggestion
}

```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

File: public/data/ahmedsilat44.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://github.com/ahmedsilat44",
+      "icon": "github"
+    },
+    {
+      "name": "",
+      "url": "",
+      "icon": ""
+    }
+  ],
+  "milestones": [
```
[the remark this answers is not in the document]
Comment (ahmedsilat44) [reply public/data/ahmedsilat44.json:18]: Ok ill remove them 👍

**Comment by ahmedsilat44** [comment]
@Panquesito7 Hey, I made the changes. Pls review it and tell me if it looks good. Thanks 😄

**Review by Panquesito7 — APPROVED** [review]
Looks great. Thanks for contributing! 🎉

**Pull request merged**